fenix/app/src/test/java/org/mozilla/fenix/shopping/ReviewQualityCheckFeatureTest.kt

```diff
@@ -205,6 +220,31 @@ class ReviewQualityCheckFeatureTest {
             assertEquals(1, availabilityCount)
             assertFalse(availability!!)
         }
+
+    @Test
+    fun `WHEN the shopping sheet is collapsed THEN the collapsed callback is called`() {
```

Comment:
Nice! 

Let's add another test case where we assert `onBottomSheetCollapsed` is not called when expanded is true. Right now, If we remove the `!expanded` check, none of the tests fail, the other test would catch that case.